lib/module.js

```diff
@@ -74,6 +74,22 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
+Object.defineProperty(
+  module.exports,
+  'builtinModules',
```

Comment:
bikeshed nit: `s/builtinModules/builtin` (feel free to ignore)

Reply:
I think this should definitely be plural, with or without `Modules`